src/matrix/kaldi-matrix.cc

```diff
@@ -1105,6 +1105,17 @@ void MatrixBase<Real>::GroupPnormDeriv(const MatrixBase<Real> &input,
         (*this)(i, j) = (input_val == 0 ? 0 : (input_val > 0 ? 1 : -1));
       }
     }
+  } else if (power == Real(1.0/0.0)) {
```

Comment:
This construct previously caused problems on a Windows compiler, you should use std::numeric_limits<Real>::infinity()